client/src/pages/ProfileListing/CoffeCard.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';

```

Comment:
Should not be using these components, this is v4 of Material UI. Please use the latest Material UI instead.